### DISCUSSION TITLE
Interp with different dimensions

### DISCUSSION THREAD

**Discussion by JordenGershenson** [pydata/xarray#8397] [Office Hours]
Hi all,

I'm somewhat new to GIS and Xarray. I'm working on trying to get a CALIOP (LEO) data array into a GOES Radiance (L1B RadF) DataSet. 

I've got both of them with lat/lon coords (not dimensions) but can't seem to get them to resample/match. I'm a little lost and can't help but feel there's an easier way to do this. Does anyone have experience with this type of problem?

I would include some code, but it's kind of a mess at the moment and I'm just looking to see if anyone is familiar with this sort of thing.

Thanks!

Jorden

**Comment by scottyhq** [comment]
Hi @CouncilFox ! Thanks for the question. If you are working with combining geo-referenced satellite data and reprojections I'd encourage you to check out https://corteva.github.io/rioxarray/stable/examples/reproject_match.html 

Please do include code (no worries about messiness) either in a comment or a [gist](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists#about-gists) as otherwise it's hard to know how to help. And note rioxarray also has a discussion forum (https://github.com/corteva/rioxarray/discussions) where you might get more assistance with specific examples.
